### ISSUE TITLE
Splitting code into functions

### ISSUE THREAD

**Issue by zurda** [fac-15/hungryish-app#31]
Reading the code in `main.js`, you might consider splitting the code into functions. This could help readability and maintainability, and assure you're following the DRY principle. For example, grouping code that manipulates the dom into a function, etc

**Comment by Oliversw** [comment]
Agreed! We were hoping to do so but were a little lost in thinking about how we could do it. Very inspired by the solution in your code though!
